src/Components/Issue/Issue.js

```diff
@@ -1,32 +1,39 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 
 const Issue = (props) => {
-  const { issue: { id, body }, toggleIssueCompletion, handleBodyChange, removeIssue, completed } = props;
+  const {
+    issue: { id, body }, location: { pathname }, 
+    toggleIssueCompletion, handleBodyChange, removeIssue, completed, location, noteId } = props;
   return (
-    <li key={id} id={id} className='issue-li'>
-      <span>
-        <button 
+    <li key={id} id={id} className="issue-li">
+      <div>
+        <button
           onClick={toggleIssueCompletion}
           className={completed ? 'complete-item' : 'incomplete-item'}
         />
-        <input
-          onChange={handleBodyChange}
-          placeholder='Add a list item...'
-          value={body}
-          maxLength='45'
-        />
-      </span>
-      <button onClick={removeIssue} className='list-delete-button'/>
+        {(pathname === `/notes/${noteId}` || pathname === '/new-note') && (
```

Comment:
This checks the pathname and will only render the input element if the pathname is new-note or a note-id